.mise.toml

```diff
@@ -0,0 +1,20 @@
+[env]
+
+[tools]
+github-cli = "latest"
+"go:github.com/go-delve/delve/cmd/dlv" = "latest"
+"go:golang.org/x/tools/gopls" = "latest"
+go = "1.24"
+gofumpt = "latest"
+golangci-lint = "1.64.2"
+jq = "latest"
+node = "20"
+"npm:pnpm" = "latest"
+"npm:typescript" = "latest"
+"npm:typescript-language-server" = "latest"
+"npm:yarn" = "latest"
+python = "3.11"
```

Comment:
Is this required to build Java?

Reply:
I've been a bit cautious since the `pulumi` submodule means that in theory you may be dipping in and out of the "full monorepo" at points and it's (IMO) handy to have a superset of that tooling available. But I'm happy if we think this is a bad call.

Reply:
Shoudl we add `dotnet` too in that case?  I'm fine either way to be clear, happy to get this in eithre way.